packages/manager/apps/dedicated/client/app/expired/expired.routing.js

```diff
@@ -0,0 +1,18 @@
+import get from 'lodash/get';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.expired', {
+    url: '/expired',
+    params: {
+      error: null,
+    },
+    component: 'managerErrorPage',
```

Comment:
I'm having trouble understanding the difference with the error page here

Reply:
This is the same as the error page, but with the sidebar displayed :)